src/python/dxpy/nextflow/ImageRefFactory.py

```diff
@@ -35,7 +35,7 @@ def get_image(self):
         return image(
             process=self._image_ref["process"],
             digest=self._image_ref["digest"],
-            dx_file_id=self._image_ref.get("dx_file_id", None),
+            dx_file_id=self._image_ref.get("file_id", None),
```

Comment:
is the image ref dict in nextaur also refactored?

Reply:
it was misnamed here. No need